web/src/pages/Dashboard/index.tsx

```diff
@@ -32,27 +112,34 @@ export default function Dashboard() {
           <div id="IPCA">
             <h1>Títulos indexados ao IPCA</h1>
             <div id="card-list">
-              <Card />
-              <Card />
-              <Card />
-              <Card />
-              <Card />
-              <Card />
+              {bonds &&
+                [...bonds.IPCA, ...bonds.IGPN].map(bond => (
+                  <Card key={bond.id} {...bond} />
+                ))}
```

Comment:
(See comments for lines below before this one)
You can actually leave this IGP-M map here if you think this won't change the behavior of the application if there are no IGP-M bonds to be listed (in the market, today, there aren't any), and that it won't break if there are any.

Reply:
Well there was one yesterday you twat

Reply:
There's only one and he's not being traded anymore

Reply:
E vc acha q eu ia sabe disso como? Eu vejo na API eu coloco no painel 🔧

Reply:
literalmente estou te falando!